### ISSUE TITLE
Failing tests in CI

### ISSUE THREAD

**Issue by hololeap** [jgm/citeproc#81]
I see a bunch of failing tests [here](/jgm/citeproc/runs/3098221409). I'm also noticing similar results when I try to build `citeproc-0.4.1` in Gentoo.

**Comment by hololeap** [comment]
I just found https://github.com/jgm/citeproc#known-bugs-and-limitations
which explains the failing tests.

I am still curious if this test failures in pandoc are related. (jgm/pandoc#7449)

**Comment by jgm** [comment]
Correct. CI passes when we pass all the tests we expect to pass, given the "known bugs and limitations."
